client/my-sites/stats/summary/index.jsx

```diff
@@ -142,6 +142,20 @@ class StatsSummary extends Component {
 							summary
 							listItemClassName={ listItemClassName }
 						/>
+						<div className="stats-module__footer-actions--summary-tall">
+							<UpsellNudge
+								title={ translate( 'Add Google Analytics' ) }
+								description={ translate(
+									'Upgrade to a Premium Plan for Google Analytics integration.'
+								) }
+								event="googleAnalytics-stats-countries"
+								feature={ FEATURE_GOOGLE_ANALYTICS }
+								plan={ PLAN_PREMIUM }
+								tracksImpressionName="calypso_upgrade_nudge_impression"
+								tracksClickName="calypso_upgrade_nudge_cta_click"
+								showIcon={ true }
+							/>
+						</div>
```

Comment:
It makes sense that the summary page bundle would go up in size once the UpsellNudge was moved from the traffic page component to the summary page component.